### DISCUSSION TITLE
How to download a full directory in sftp ?

### DISCUSSION THREAD

**Discussion by romainlavisse** [Eugeny/tabby#5419] [Q&A] [answered]
HI,
how can i download a full directory in sftp mode when im connected to my server in ssh mode ?

**Comment by alanyoungCN** [comment]
I wonder, too.
@Eugeny cc plz

**Reply by Eugeny** [reply]
Directory up/downloads are planned but not implemented yet.

**Reply by alanyoungCN** [reply]
Thanks for reply. Looking forward to this feature.

**Comment by yangyongyongyong** [comment]
2023.2.13  no download dir

**Comment by JunerLee** [comment]
2023.5.6 no download dir

**Comment by dudebing99** [comment]
2023.5.20 not yet

**Comment by yangyongyongyong** [comment]
2023.6.21 not yet

**Reply by Eugeny** [reply, marked as answer]
Honestly this is the most disrespectful shit I've seen here.

**Reply by creeponsky** [reply]
> Honestly this is the most disrespectful shit I've seen here.

Hi, no offense, just wondering how things are going, if it hasn't started yet, can I try and see if there's anything I can do

**Reply by Eugeny** [reply]
Honestly that would be amazing. Another contributor has already stepped in to implement folder uploads in https://github.com/Eugeny/tabby/pull/9891, but folder downloads are still up for grabs.

**Comment by lu-pengju** [comment]
2024 11.3 not yet

**Comment by emoxam** [comment]
2 year gone, no news..

**Comment by usshauler** [comment]
Can we please get someone to implement this feature ?

**Comment by apxangels** [comment]
can we have download by sftp?
